### PR TITLE
create org update req

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

File: src/service/iot_config.proto [hunk 1]
```diff
@@ -157,6 +157,16 @@ message org_create_roamer_req_v1 {
   repeated bytes delegate_keys = 6;
 }
 
+message org_update_req_v1 {
+  uint64 oui = 1;
+  oneof update {
+    bytes payer = 2;
+    repeated bytes delegate_keys = 3;
```
Comment (jeffgrunewald) [inline src/service/iot_config.proto:164]: would you simply overwrite the list of delegate keys each time or does this need an `action` attribute as well to be able to add/remove delegate keys?

Comment (macpie) [reply src/service/iot_config.proto:164]: Was thinking of overwrite

**Comment by macpie** [comment]
We should also think of adding a `RPC`  to handle adding or removing DevAddr ranges on an org

**Comment by jeffgrunewald** [comment]
handled by https://github.com/helium/proto/pull/326

**Pull request closed without merging**